lib/src/fnc/duration.rs

```diff
@@ -0,0 +1,63 @@
+use crate::err::Error;
+use crate::sql::duration::Duration;
+use crate::sql::value::Value;
+
+pub fn secs((duration,): (Option<Value>,)) -> Result<Value, Error> {
+	let duration = match duration {
+		Some(Value::Duration(d)) => d,
+		None => Duration::default(),
```

Comment:
IMO the `Duration` shouldn't be `Option`al like `Datetime` because the default of "zero" is considerably less useful than "now."

Reply:
That actually makes a lot of sense and I do agree with you that defaulting to `0ns` is probably not what we would want.